src/Simplex/Chat/Library/Commands.hs

```diff
@@ -308,19 +308,20 @@ processChatCommand cmd =
 processChatCommand' :: VersionRangeChat -> ChatCommand -> CM ChatResponse
 processChatCommand' vr = \case
   ShowActiveUser -> withUser' $ pure . CRActiveUser
-  CreateActiveUser NewUser {profile, pastTimestamp} -> do
+  CreateActiveUser NewUser {profile, pastTimestamp, userSuperpeer} -> do
```

Comment:
I don't think NewUser/CreateActiveUser should allow super peer - there is another api to create local super peer.

Reply:
I created separate api type initially but almost all code could be reused so I changed it to parameter. There's still separate api string for CLI